include/openenclave/host.h

```diff
@@ -14,6 +14,7 @@
 #error "enclave.h and host.h must not be included in the same compilation unit."
 #endif
 
+#include <openenclave/bits/edl/asym_keys.h>
```

Comment:
Does the `bits/edl` folder have to be installed too?

Reply:
Yes, it's installed by https://github.com/openenclave/openenclave/blob/74c7a6f9f93bff670c2e6d5ec0548552da5012c8/common/edl/CMakeLists.txt#L38

Reply:
```
install(DIRECTORY ${OE_EDL_INCLUDES}
  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/bits)
```

Isn't it getting installed into `openenclave/bits/`? Should the above instead be:

```
install(DIRECTORY ${OE_EDL_INCLUDES}
  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/bits/edl)
```

Either that or `set(OE_EDL_INCLUDES ${OE_INCDIR}/openenclave/bits/edl)` should be `set(OE_EDL_INCLUDES ${OE_INCDIR}/openenclave/bits)`

Reply:
Blocking on this because I think it will break when building a sample or something outside the OESDK tests.

Reply:
The `DIRECTORY` keyword works as copying the entire directory to the destination.
In this case, copy the `edl` directory to `${CMAKE_INSTALL_INCLUDEDIR}/openenclave/bits`. The result is creating `${CMAKE_INSTALL_INCLUDEDIR}/openenclave/bits/edl`.

Reply:
Oh ok, thanks for the clarification :smile:

Reply:
I found that because I was specifying exactly `${CMAKE_INSTALL_INCLUDEDIR}/openenclave/bits/edl' and samples test failed.

Reply:
Side note: Is there a reason for partitioning this out into the edl subfolder when publishing to  bits? i.e. at the point of consumption in the SDK package, whether this header was edl generated or not is irrelevant

Reply:
Agree. Will directly publish headers to bits instead.